bin/all_sky_search/pycbc_fit_sngls_by_template

```diff
@@ -176,7 +176,11 @@ abovethresh, stat = get_stat(args, trigf[args.ifo], args.stat_threshold)
 tid = trigf[args.ifo + '/template_id'][abovethresh]
 time = trigf[args.ifo + '/end_time'][abovethresh]
 if args.save_trig_param:
-    tparam = trigf[args.ifo + '/' + args.save_trig_param][abovethresh]
+    tparam_region = trigf[args.ifo][args.save_trig_param + '_template'][:]
+    # Note: this breaks if there are no triggers in the template, but that
```

Comment:
Though I have pointed out a case where this can break, it is better than before, as it was breaking if there were no triggers _above threshold_ in the template, rather than no triggers at all

Reply:
OK ... but how about putting a try-except to catch the no triggers case?

Reply:
Added now